charts/prometheus/templates/server/deploy.yaml

```diff
@@ -9,6 +9,9 @@ metadata:
 {{- end }}
   labels:
     {{- include "prometheus.server.labels" . | nindent 4 }}
+{{- if .Values.server.deploymentLabels }}
```

Comment:
Imho this should be named extra labels and added to all resources.

Reply:
Yes, for example the node_exporter daemonset can't be labeled either.